modules/my_health/app/controllers/my_health/v1/prescription_documentation_controller.rb

```diff
@@ -6,7 +6,18 @@ class PrescriptionDocumentationController < RxController
       def index
         if Flipper.enabled?(:mhv_medications_display_documentation_content, @current_user)
           begin
-            documentation = client.get_rx_documentation(params[:ndc])
+            id = params[:id]
+            rx = client.get_rx_details(id)
+            raise StandardError, 'Rx not found' if rx.nil?
+
+            cmop_ndc_number = if rx[:rx_rf_records]&.[](0)&.[](1)&.[](0)&.key?(:cmop_ndc_number)
+                                rx[:rx_rf_records][0][1][0][:cmop_ndc_number]
+                              elsif rx[:cmop_ndc_number].present?
+                                rx[:cmop_ndc_number]
+                              end
```

Comment:
Could you use dig and presence here?

```suggestion
            cmop_ndc_number = rx.dig(:rx_rf_records, 0, 1, 0, :cmop_ndc_number) || rx[:cmop_ndc_number].presence

```

Reply:
I had to change it slightly. `rx_rf_records` was the array, dig doesn't work on custom objects. I also changed the logic to find any value in the array instead of just at the 0 index. Just mentioning to let you know it was intended. 

@ericboehs

Reply:
@ericboehs I had to update it one more time, took me a minute to understand how dig works. Could you re-approve when you have a chance? Ty! and thanks for the suggestion.